README.md

```diff
@@ -51,12 +55,15 @@ Simply copy the **menas.war** file produced when building the project into Tomca
 #### Standardization and Conformance requirements:
 - **Spark 2.4.0** installation
 - **Hadoop 2.7** installation
+- **Menas** running instance
```

Comment:
`**Menas** running instance with models with specific data filled out`

Or something similar. So it doesn't seem like you just need to deploy it to tomcat, but it is actively used.

Reply:
I kind of feel like that is already explained in the `Menas` section, as well as in the `Standardization` and `Conformace` sections under `What is Enceladus?`.

Reply:
Ok, I might be too strict in this.